pkg-manager/core/src/install/index.ts

```diff
@@ -749,8 +749,17 @@ function getOutdatedLockfileSetting (
   return null
 }
 
+export function sortObjectKeys<Obj> (obj: Obj): Obj {
+  if (typeof obj !== 'object' || obj === null || Array.isArray(obj)) return obj
+  const entries = Object
+    .entries(obj)
+    .sort(([k1, _v1], [k2, _v2]) => k1.localeCompare(k2, 'en'))
+    .map(([k, v]) => [k, sortObjectKeys(v)])
+  return Object.fromEntries(entries) as Obj
+}
+
 export function createObjectChecksum (obj: unknown) {
-  const s = JSON.stringify(obj)
+  const s = JSON.stringify(sortObjectKeys(obj))
```

Comment:
use the sort-keys library. It is already used in other places.

Reply:
How this fix this error when test?

```text
SyntaxError: Cannot use import statement outside a module
```